src/Compilers/CSharp/Test/Emit/Attributes/InternalsVisibleToAndStrongNameTests.cs

```diff
@@ -1321,13 +1423,31 @@ public void SignModuleKeyFileCmdLine_1()
             var x = s_keyPairFile;
             string s = String.Format("{0}{1}{2}", @"[assembly: System.Reflection.AssemblyKeyFile(@""", x, @""")] public class C {}");
 
-            var other = CreateStandardCompilation(s, options: TestOptions.ReleaseModule.WithCryptoKeyFile(s_keyPairFile).WithStrongNameProvider(s_defaultProvider));
+            var options = TestOptions.ReleaseModule.WithCryptoKeyFile(s_keyPairFile).WithStrongNameProvider(s_defaultPortableProvider);
+            var other = CreateStandardCompilation(s, options: options);
+
+            var outStrm = new MemoryStream();
+            var success = other.Emit(outStrm);
+            Assert.True(success.Success);
+
+            ConfirmModuleAttributePresentAndAddingToAssemblyResultsInSignedOutput(outStrm, AttributeDescription.AssemblyKeyFileAttribute, legacyStrongName: false);
+        }
+
+        [WorkItem(531195, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/531195")]
+        [Fact()]
+        public void SignModuleKeyFileCmdLine_1_Bypass()
```

Comment:
Perhaps rename this `_Legacy` since it does not use the `Feature` option, and for consistency with other tests.

Reply:
Done.